package.json

```diff
@@ -38,8 +38,10 @@
   },
   "homepage": "https://github.com/octopusthink/nautilus#readme",
   "dependencies": {
+    "feather-icons": "^4.22.1",
     "invariant": "^2.2.4",
     "prop-types": "^15.7.2",
+    "react-inlinesvg": "^0.8.4",
```

Comment:
We're just creating the SVG ourselves (which is easier/safer), so we don't need this. I'll remove it 😄